plugins/forwarder/grpc/nativemeter/forwarder.go

```diff
@@ -98,22 +98,28 @@ func (f *Forwarder) Forward(batch event.BatchEvents) error {
 			}
 		}
 	}()
+	oldTransformDataCount := 0
 	for _, e := range batch {
-		if data, ok := e.GetData().(*v1.SniffData_Meter); ok {
-			if err := f.handleMeter(data, streamMap); err != nil {
-				return err
-			}
-		}
-		if data, ok := e.GetData().(*v1.SniffData_MeterCollection); ok {
+		switch data := e.GetData().(type) {
+		case *v1.SniffData_MeterCollection:
 			if err := f.handleMeterCollection(data, streamMap); err != nil {
 				return err
 			}
+		case *v1.SniffData_Meter:
+			oldTransformDataCount++
```

Comment:
As we have moved to `MeterCollection`, let's ignore meter(s) in queue(mmap) directly.

Reply:
updated.